### ISSUE TITLE
Communities Of Practice Page: Update alt text for CoP page card images to adhere to WCAG

### ISSUE THREAD

**Issue by MattPereira** [hackforla/website#3860]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `Community of Practice` page so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [ ] Change the image alt property value within `/pages/communities-of-practice.html`:

**From:**
```
alt="{{ community[1].alt }}"
```

**To:**
```
alt=""
```

- [ ] Ensure that the corresponding `Community of Practice` webpage stays the same after the change
- [ ] Using developer tools to inspect the image, ensure that the new alt text gets incorporated after the change.
  - **Note:** `alt=""` in code while show up as `alt` when using developer tools to inspect the image's alt text property.

### Resources/Instructions
- Communities of Practice webpage: https://www.hackforla.org/communities-of-practice

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `/pages/communities-of-practice.html`
- Click on the filename below to see the source file code:
https://github.com/hackforla/website/blob/910e145af61f4072aa8ebb24da385197729af942/pages/communities-of-practice.html#L31

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by ExperimentsInHonesty** [comment]
@EllenKellyb I assume you are still working on this issue since you are still assigned.  So I am moving it to the `In Progress` column

**Comment by EllenKellyb** [comment]
i. Availability: Weeknights 7-9 pm
ii. ETA: Edited the page per instruction, Created the pull request on 3/23/23.
